### PR TITLE
Adding mode to alert's LogHandler

### PR DESCRIPTION
**Pull request by yosiat**
Issue - #368

### CODE REVIEW

File: pipeline/alert.go [hunk 1]
```diff
@@ -428,11 +428,22 @@ type ExecHandler struct {
...
+//    stream
+//         |alert()
+//					.log('/tmp/alert')
+//					.mode(0644)
 // tick:property
 func (a *AlertNode) Log(filepath string) *LogHandler {
 	log := &LogHandler{
 		AlertNode: a,
 		FilePath:  filepath,
+		Mode:      0600,
```
Comment (nathanielc) [inline pipeline/alert.go]: We should define this as a constant `Default LogFileMode` or something.

**Comment by nathanielc** [comment]
@yosiat Overall looks great, thanks for the PR.

**Comment by yosiat** [comment]
@nathanielc Fixed :+1:

File: pipeline/alert.go [hunk 2]
```diff
@@ -16,6 +16,9 @@ const defaultMessageTmpl = "{{ .ID }} is {{ .Level }}"
 // Default template for constructing a details message.
 const defaultDetailsTmpl = "{{ json . }}"
 
+// Default log mode for file
+const defaultLogFileMode = 0666
```
Comment (nathanielc) [inline pipeline/alert.go]: We still want a default of `0600` right?

Comment (yosiat) [inline pipeline/alert.go]: oops :(

**Comment by eran-totango** [comment]
@yosiat Hey, this doesn't seem to work.
Kapcitor creates the alert log file with different permissions that what i tried to set.

**Comment by lafrech** [comment]
I just used this and it works.

Here's the proper syntax, applied to an example from the docs:

```
dbrp "telegraf"."autogen"

stream
    // Select just the cpu measurement from our example database.
    |from()
        .measurement('cpu')
    |alert()
        .crit(lambda: int("usage_idle") < 100)
        // Whenever we get an alert write it to a file.
        .log('/tmp/alerts.log')
        .mode(0644)
```

**Pull request merged**